src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -241,6 +241,8 @@ public function listAction(Request $request, $limit = 10, $offset = 0, $orderBy
         $adminProductWrapper = $this->container->get('prestashop.adapter.admin.wrapper.product');
         $totalCount = 0;
 
+        $this->get('prestashop.service.product')->cleanupOldTempProducts();
```

Comment:
A model class is not a service. More, ``getOldTempProducts`` is a static function :)

How about ``\Product::cleanupOldTempProducts();`` ?